teamwork/apps/profiles/views/EditProfileView.py

```diff
@@ -163,13 +181,17 @@ def edit_profile_helper(request, username):
     else:
         #grab profile for the current user
         profile = Profile.objects.get(user=request.user)
+        tempProfile = User.objects.get(username=username)
```

Comment:
Change this line to 

```profileUser = User.objects.get(username=profile.username)```

Username is the page user and as a result, you would be able to change any users email that is why it is in the Admin(isGT) if

Reply:
`tempProfile = User.objects.get(username=username)` will allow any user to visit another user's profile and edit their email address. This should be `tempProfile = User.objects.get(username=profile.username)`.